source/requirements.txt

```diff
@@ -9,3 +9,4 @@ sphinx-notfound-page==0.4
 transifex-client==0.13.9
 sphinx-intl==2.0.1
 sphinxcontrib-svg2pdfconverter==1.0.1
+-e git+https://github.com/readthedocs/sphinx-hoverxref.git@3558a67e56bf27f08d52425bbcd1d63989c6224e#egg=hoverxref.extension
```

Comment:
Do they not work with the latest version?

Reply:
It's not on pypi yet.

Reply:
https://pypi.org/project/sphinx-hoverxref/#files

Reply:
They haven't published a release with term support yet.